data/import_data.py

```diff
@@ -0,0 +1,131 @@
+import yaml
+import os
+
+from app import db  # noqa: F401
+from models import (
+    Category,
+    Criterion,
+    HonorableMention,
+    InnovativePolicyIdea,
+    Score,
+    State,
+    StateGrade,
+    StateCategoryGrade,
+    ResourceLink
+)
+from tests.test_utils import clear_database
+
+SCRIPT_DIR = os.path.dirname(__file__)
+
+
+def import_data():
+    print('Clearing database')
+    clear_database(db)
+
+    print('Importing categories')
+    import_categories()
+
+    for filename in os.listdir(absolute_file_path('states')):
+        state = filename.split('.yaml')[0]
+        print(f'Importing state: {state}')
+
+        import_state(absolute_file_path(f'states/{filename}'))
+
+
+def absolute_file_path(file_name):
+    return os.path.join(SCRIPT_DIR, file_name)
+
+
+def import_state(path):
```

Comment:
This method imports information about an individual state, including the state code and name, the overall grade, the grades for each category, and whether the state meets all the criteria.